Dockerfile

```diff
@@ -96,7 +96,9 @@ RUN apt-get update -q --fix-missing && \
   touch /var/log/auth.log && \
   update-locale && \
   rm -f /etc/cron.weekly/fstrim && \
-  rm -f /etc/postsrsd.secret
+  rm -f /etc/postsrsd.secret && \
+  rm -f /etc/postfix/dhparams.pem && \
```

Comment:
why delete dhparams.pem and dh.pem, when they are only generated later on?

Reply:
To make sure there are no default files left over from the installation. The start-mailserver.sh script only generates the files if they don't exist so I want to make sure that they are gone when the container starts the first time. I'm not sure, but if I remember correctly at least dh.pem exists in a default version after a clean installation. Better safe than sorry.

Reply:
>  Better safe than sorry.

yes, that could be a point.

But 

Just to test I just removed the -f from the rm command (which will make rm error out if the to be deleted file is not there) and it did through an error. which means the file is not there at this stage in the build.

Reply:
Right. I'll remove the unnecessary deletes then. I don't have access to my Linux environment for a few days, so it will take some time.

Reply:
If you like I could also do this modification (if you enabled that Maintainers may edit your pr) or fix this and do a manual merge.

Reply:
I think I enabled that, I usually do. Please go ahead, otherwise I'll fix it when I get back.